components/button/stories/button.stories.js

```diff
@@ -82,14 +82,31 @@ export default {
 		},
 		staticColor: {
 			name: "Static color",
+			description: "Variants that can be used when a button needs to be placed on top of a colored background or a visual.",
 			type: { name: "string" },
 			table: {
 				type: { summary: "string" },
 				category: "Advanced",
 			},
 			options: ["white", "black"],
 			control: "select",
-		}
+		},
+		showIconOnlyButton: {
+			table: {
+				disable: true,
+			},
+		},
+		layout: {
+		    name: "Layout",
+		    description: "How the buttons align in the preview (Storybook only).",
```

Comment:
Just noting here that this will probably no longer be necessary after the storybook layout decorator merges - plus side, I might steal this approach for that PR #2420!